skfda/exploratory/visualization/representation.py

```diff
@@ -184,21 +219,18 @@ def __init__(
         else:
             self.gradient_list = []
```

Comment:
Why empty list instead of `None`?

Reply:
because then we check the length

Reply:
You can check if it is `None` instead, and probably is more clear that way.

Reply:
ok